build.md

```diff
@@ -155,6 +155,39 @@ args:
   - GIT_COMMIT
 ```
 
+### ssh
+
+`ssh` defines an SSH authentication that the image builder SHOULD use during image build (e.g., cloning private repository)
+
+`ssh` property syntax can be either:
+* `default` - let the builder connect to the ssh-agent.
```

Comment:
`default` might not be a very obvious name, maybe make it more explicit as `ssh-agent`?

Reply:
afaik `default` also has to be supplied to the `--ssh` flag of Docker.

Reply:
indeed. We can define an alias then (as we do for `pull: ("always"|"missing"|"never")` for better compatibility with docker CLI

Reply:
otherwise, I'm also fine we just use `default`, no strong opinion here